dbt_subprojects/tokens/models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1877,8 +1877,11 @@ FROM
     ('deusd-elixir-deusd','ethereum', 'DEUSD', 0x15700b564ca08d9439c58ca5053166e8317aa138, 18),
     ('unibot-unibot','ethereum', 'UNIBOT', 0xf819d9cb1c2a819fd991781a822de3ca8607c3c9, 18),
     ('morpho-morpho','ethereum', 'MORPHO', 0x58d97b57bb95320f9a05dc918aef65434969c2b2, 18),
-    ('scrvusd-savings-crvusd', 'ethereum', 'SCRVUSD', 0x0655977FEb2f289A4aB78af67BAB0d17aAb84367, 18)
-
+    ('scrvusd-savings-crvusd', 'ethereum', 'SCRVUSD', 0x0655977FEb2f289A4aB78af67BAB0d17aAb84367, 18),
+    ('trx-tron', 'ethereum', 'TRON', 0x50327c6c5a14dcade707abad2e27eb517df87ab5, 6),
+    ('usdc-usdc', 'ethereum', 'aEthUSDC', 0x98c23e9d8f34fefb1b7bd6a91b7ff122f4e16f5c, 6),
```

Comment:
using usd price for aUSDC.
although aave USDC is a rebasing token, it is always 1:1 USDC.